Add doc about dry-run option for add-missing-* cmd (#8084)
nextcloud/documentation: admin_manual/maintenance/upgrade.rst

 $ sudo -u www-data php occ db:add-missing-columns
 $ sudo -u www-data php occ db:add-missing-indices
 $ sudo -u www-data php occ db:add-missing-primary-keys

> You can use the ``--dry-run`` option have the commands output the SQL queries instead of running them.

Reviewer: ```suggestion
You can use the ``--dry-run`` option to output the SQL queries instead of executing them.
```

Author: Yep, yours is way less complex :laughing:
